esp/src/src-react/layouts/HolyGrail.tsx

```diff
@@ -23,6 +23,6 @@ export const HolyGrail: React.FunctionComponent<HolyGrailProps> = ({
             <div style={{ flex: "1 1 auto", minWidth: 1, minHeight: 1 }}>{main}</div>
             <div style={{ flex: "0 2" }}>{right}</div>
         </div>
-        <footer style={{ flex: "0 0", minWidth: 0 }}>{footer}</footer>
+        <footer>{footer}</footer>
```

Comment:
Is this safe?  We could make the footerStyle an optional property which defaults to this?